### ISSUE TITLE
Visualize ladder progress (Mock-up pic)

### ISSUE THREAD

**Issue by narucy** [Epix37/Hearthstone-Deck-Tracker#1787]
I want to check my ladder run good (or bad) and I try visualize export CSV file and using google but It's a little bit boring work. I hope HDT built-in feature (some poker tracker style) line-graph.

![ladder_graph fw](https://cloud.githubusercontent.com/assets/4232498/11397097/30f263b2-93ba-11e5-84f1-2159685966a5.png)

**Comment by Cu8eR** [comment]
+1

**Comment by culinko** [comment]
I love this graph! Really nice idea.

I was also thinking about some kind of "Season recap" pages, which would show you some stats in overview: Season number, ending rank, hours played, best hero, most played hero, rewards (season 17+). So technically you can have even more graphs, like ending rank (comparing the seasons) or time played (comparing the seasons, within one season), etc.

(Unfortunately, for ending rank to work automatically, the rank detection would have to improve, so it would be reliable and also be able to track either both of your ranks (pre-game and post-game) or track your stars as well).

**Comment by RodrigoMoyses** [comment]
+1 all of those would be great aditions!

**Comment by Epix37** [comment]
> Unfortunately, for ending rank to work automatically, the rank detection would have to improve, so it would be reliable and also be able to track either both of your ranks (pre-game and post-game) or track your stars as well

Yes, this is the main problem right now, working on fixing that. When that's all done, this is certainly a good idea. (so is the season recap)

**Comment by GiggliG** [comment]
For season recap. you need to account for difference between time zone of the computer running HDT/HS, and the timezone of the server.

I play in both NA and EU, and HDT considers games that I play now in NA (after midnight, 1st December) to be in the december season, when they are actually in the November season. But games I play now in the EU server are actually in the December season. However, the December season starts at 11PM on 30th November, in my time zone, so if if I played games between 11 and midnight, it would consider them as part of the November season...

Time zones are tricky!

**Comment by culinko** [comment]
Rank detection was improved and works 100% for me. This means deck tracker will know your highest and ending ranks now. Also, EndOfGame.log contains stuff that is logged every time you rank up, so this could also be used if it proves to be reliable.
